pkg/server/datastore/sqlstore/migration.go

```diff
@@ -36,7 +138,7 @@ func migrateDB(db *gorm.DB, dbType string, disableMigration bool, log logrus.Fie
 		return sqlError.New("current migration code not compatible with current release version")
 	}
 
-	isNew := !db.HasTable(&Bundle{})
+	isNew := !db.HasTable(&Migration{})
```

Comment:
It feels conceptually cleaner to rely on the migration table for this check instead of bundle, which is now possible since any upgrade path to this code would have created it already. When this code was first added, the migration table was being added at the same time, so we needed something else.